src/index.js

```diff
@@ -65,14 +63,14 @@ let fetchSetter = {
 }
 
 function findAmountSpentOnAYear(totapTrips) {
-        let totalSpentForATrip = totapTrips.reduce((totalPrice, trip) => {
-            let costPerDuration = (trip.estimatedLodgingCostPerDay * trip.duration);
-            let totalPricePerPerson = (costPerDuration += trip.estimatedFlightCostPerPerson);
-            let totalPriceForTheTrip = (totalPricePerPerson * trip.travelers)
-            totalPrice += totalPriceForTheTrip;
-            return totalPrice
-        }, 0)
-        domUpdates.tellMeYourMoneys(totalSpentForATrip * 1.1)
+    let totalSpentForATrip = totapTrips.reduce((totalPrice, trip) => {
+        let costPerDuration = (trip.estimatedLodgingCostPerDay * trip.duration);
+        let totalPricePerPerson = (costPerDuration += trip.estimatedFlightCostPerPerson);
```

Comment:
```suggestion
        let totalPricePerPerson = (costPerDuration + trip.estimatedFlightCostPerPerson);
```

We don't need to amend costPerDuration so we can just use `+` instead of `+=`

Reply:
Hmm, I thought I had commit this suggestion last time. Maybe It's because I'm not pulling down from main every time, because it's only me. I'll make this change right now in a clean up branch.